pages/collective-page.js

```diff
@@ -58,14 +60,20 @@ class CollectivePage extends React.Component {
     }
 
     let skipDataFromTree = false;
+    let ssrGraphqlError;
 
     // If on server side
     if (req) {
-      await preloadCollectivePageGraphlQueries(slug, client);
-      skipDataFromTree = true;
+      try {
+        await preloadCollectivePageGraphlQueries(slug, client);
+        skipDataFromTree = true;
+      } catch (e) {
+        console.error(`Collective page: unable to preload data for ${slug}`, e);
+        ssrGraphqlError = e;
```

Comment:
Had to come up with this workaround because of https://github.com/apollographql/apollo-client/issues/4644